wikipedia-sentences/README.md

```diff
@@ -14,14 +16,14 @@
       Output
     </td>
     <td>
-      <pre>top_k</pre> number of sentences that match input query
+      <i>top_k</i> number of sentences that match input query
```

Comment:
We can use markdown table here maybe? Like using this https://www.tablesgenerator.com/markdown_tables